hack/local-up-cluster.sh

```diff
@@ -122,7 +122,14 @@ fi
 
 # set feature gates if using ipvs mode
 if [ "${KUBE_PROXY_MODE}" == "ipvs" ]; then
-    FEATURE_GATES="$FEATURE_GATES,SupportIPVSProxyMode=true"
+    # If required kernel modules are not available, fall back to iptables.
+    sudo modprobe -a ip_vs ip_vs_rr ip_vs_wrr ip_vs_sh nf_conntrack_ipv4
+    if [[ $? -eq 0 ]];
+    then
+      FEATURE_GATES="$FEATURE_GATES,SupportIPVSProxyMode=true"
+    else
+      KUBE_PROXY_MODE=iptables
```

Comment:
would be helpful to log a message that we are falling back to iptables mode

Reply:
Done